whisper_live/client.py

```diff
@@ -187,15 +190,38 @@ def on_message(self, ws, message):
         # keep only last 3
         if len(text) > 3:
             text = text[-3:]
+
+        # Join the transcribed text segments into a single string
+        transcribed_text = "".join(text)
+
+        # Check if there is text in transcription before translating
+        if not transcribed_text:
+            logging.info("No text in transcription to translate.")
+            return
+
+        # Wrap the transcribed text to a width of 60 characters
         wrapper = textwrap.TextWrapper(width=60)
-        word_list = wrapper.wrap(text="".join(text))
-        # Print each line.
-        if os.name == "nt":
-            os.system("cls")
-        else:
-            os.system("clear")
+        word_list = wrapper.wrap(text=transcribed_text)
+
+        # Log each wrapped line of the transcribed text
         for element in word_list:
-            print(element)
+            logging.info(
+                colorama.Fore.RED
+                + f"Transcription: {element}"
```

Comment:
hello there this is pretty interesting!
```suggestion
                + f"Transcription: {element}"
```

Reply:
nah we up now!